webscalesqlclient/src

```diff
@@ -1 +1 @@
-webscalesql-5.6/
\ No newline at end of file
+../../build/third-party/webscalesqlclient/webscalesql-5.6
```

Comment:
This doesn't look right to me. Where does this dir come from? Why is the previous symlink incorrect?